cmake/Findamxc.cmake

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# SPDX-FileCopyrightText: 2019-2020 the prplMesh contributors (see AUTHORS.md)
+#
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+find_library(AMXC_LIBRARY "libamxc.so" PATHS ${CMAKE_PREFIX_PATH}/usr/lib/)
```

Comment:
I don't think we need PATHS here. `find_library(AMXC_LIBRARY "libamxc.so")` should work as well. The PATHS thing was introduced for a specific situation where there was some dependency that was installed in a non-default location, I think.

Reply:
Applied. Resolving.